### ISSUE TITLE
The aligned plot is not being created

### ISSUE THREAD

**Issue by JavierTG97** [uzh-rpg/rpg_trajectory_evaluation#42]
Thanks a lot for the library :)

After executing the example:
```python
python2 scripts/analyze_trajectory_single.py results/euroc_mono_stereo/laptop/vio_mono/laptop_vio_mono_MH_05
```
I got the following plot:

![image](https://user-images.githubusercontent.com/71815905/125541355-79acb256-ae99-4a8c-958e-cca32ead45fb.png)

After seing the output from the example I guess that the grountruth trajectory visually matches the aligned one. But I am not sure if the aligned trajectory has been created and taken into account to compute the error metrics.  Is there a way to check this?

Thanks in advance!

**Comment by neliudochka** [comment]
I have the same issue. Did you manage to understand if it is correct result?

**Comment by javrtg** [comment]
Hi @neliudochka, I believe that the module uses the label `'Estimate'` to refer to the aligned curve, not the (original) estimated curve. This can be seen e.g. in:

https://github.com/uzh-rpg/rpg_trajectory_evaluation/blob/8c8ceec55c5c5094a6494208cfc5f54afe0bbc4d/scripts/analyze_trajectory_single.py#L173-L177

Concretely, the label `'Estimate'` is assigned to the curve `plot_traj.p_es_aligned`. This curve results from aligning the (original) estimated curve `p_es` with the ground-truth curve `p_gt`. Here’s how it’s done:

https://github.com/uzh-rpg/rpg_trajectory_evaluation/blob/8c8ceec55c5c5094a6494208cfc5f54afe0bbc4d/src/rpg_trajectory_evaluation/trajectory.py#L250-L258

So to sum up, yes that plot is a correct (expected) result. It just has to be taken into account that the curve labeled as "Estimate" actually refers to the "Aligned" one.

I hope this helps clarify things for you!
